build/nodes-and-endpoints/join-testnet.md

```diff
@@ -1,5 +1,179 @@
+2. Verify Installations:
+	- Verify **jq**: `jq --version` (sample output: `jq-1.6`)
+	- Verify **gcc** : `gcc --version` (sample output: `gcc (Ubuntu 9.4.0-1ubuntu1~20.04.1) 9.4.0`)
+	- Verify **make**: `make --version` (sample output: `GNU Make 4.2.1`)
+	- Verify **git**: `git --version` (sample output: `git version 2.25.1`)
+
+## Installation Steps
+### Install the persistenceCore Binary
+
+{% hint style="warning" %}
+The latest version available in the [releases page](https://github.com/persistenceOne/persistenceCore/releases) might not be the one running on the test-core-1 chain. To ensure you're running the correct version, join the [Discord Testnet Validator Announcements Channel](https://discord.com/channels/796174129077813248/1042042319987294229).
+{% endhint %}
+
+1.  Clone the **persistenceCore** repository:
+	```bash
+	git clone https://github.com/persistenceOne/persistenceCore.git $GOPATH/source/persistenceCore && cd $GOPATH/source/persistenceCore
+	```
+2. Check what version is running on the test-core-1 chain by visiting the [Discord Testnet Validators Announcements Channel](https://discord.com/channels/796174129077813248/1042042319987294229).
+3. Switch to the branch of the latest version *(v6.0.0-rc5 as of time of writing)*: 
+	```bash
+	git checkout v6.0.0-rc5
+	```
+4. Install the **persistenceCore** binary:
+	```bash
+	make all
+	```
+5. Verify installation (sample output: `v6.0.0-rc5`): 
+	```bash
+	persistenceCore version
+	```
+
+### Create or Import Key (XPRT Address)
+- Create **key** *(ensure you safely store the mnemonic seed phrase)*:
+	```bash
+	persistenceCore keys add <KEY_NAME>
+	```
+- Recover **key** *(you'll be required to input your mnemonic seed phrase)*: 
+	```bash
+	persistenceCore keys add <KEY_NAME> --recover
+	```
+- Verify the **key** was created successfully:
+	```bash
+	persistenceCore keys list
+	```
+
+### Create and Sync the Node
+1. Initialize the **node** *(moniker = node name)*: 
+	```bash
+	persistenceCore init <MONIKER> --chain-id="test-core-1" # e.g. persistenceCore init "Persistence Node" --chain-id="test-core-1"
+	```
+2. Download the **test-core-1 genesis** file: 
+	```bash
+	cd ~/.persistenceCore/config && wget -O genesis.json  https://raw.githubusercontent.com/persistenceOne/networks/master/test-core-1/final_genesis.json
+	```
+3. Use **StateSync** to sync with the rest of the nodes. Follow the step-by-step guide below: 
+	- Run the following command and copy the values of `trust_height` and `trust_hash`. They are required in the next steps.
+		```bash
+		curl -s https://persistence-testnet-rpc.cosmonautstakes.com/status | jq '.result .sync_info | {trust_height: .latest_block_height, trust_hash: .latest_block_hash} | values'
+		```
+	- Open config:
+		```bash
+		nano +328 ~/.persistenceCore/config/config.toml
+		```
+	- Replace the existing settings in the opened file with the following:
+		```
+		persistent_peers = "14ecdc5126ea8d93c7d3a863d9d38e380e46fc06@185.225.233.30:26656"
```

Comment:
Add seed node, seeds are the first point of contact for a new node. They return a list of known active peers.
Detailed explanation [here](https://docs.tendermint.com/v0.34/tendermint-core/using-tendermint.html#peers).
```
[p2p]
seeds = "b4237f8a7ca357d380ad119b76cbceec7e7e8a75@seed.testnet.persistence.one:26656"
```